samples/Farfetch.LoadShedding.Samples.WebApi/Farfetch.LoadShedding.Samples.WebApi.csproj

```diff
@@ -1,9 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <NoWarn>CS1591,SA1600</NoWarn>
```

Comment:
Can we add this rule to the .editorconfig file?

Something like: 
[*.cs]
dotnet_diagnostic.CS1591.severity = none
dotnet_diagnostic.SA1600.severity = none

This way it get applied to all csproj even when new ones are created,

Reply:
It should be applied only for test projects